### PR TITLE
add issue templates

### PR DESCRIPTION
**Pull request by cliffordfajardo**
**What is the change?**
- Add issue templates

**Why are we making this change?**
- By adding [issue templates](https://help.github.com/en/articles/creating-issue-templates-for-your-repository) (a native github feature), we can prompt contributors to answer a few questions before submitting an issue.

When a user clicks `new issue` they will be prompted to choose which type of issue it is:
Screen Shot 2019-08-13 at 11 55 35 PM

After choosing the type of issue, they will be shown a template inside the issues text area editor:

Screen Shot 2019-08-13 at 11 56 54 PM

### CODE REVIEW

**Comment by terkelg** [comment]
I'm down to include a template but I think it should be changed a bit. Like the one here https://github.com/terkelg/prompts/tree/master/.github/ISSUE_TEMPLATE

**Comment by cliffordfajardo** [comment]
That's sounds reasonable 👌  I'll make changes

**Comment by cliffordfajardo** [comment]
@terkelg - Ok done. I copied the same templates from the repo you referenced
- https://github.com/terkelg/prompts/tree/master/.github/ISSUE_TEMPLATE

File: .github/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+2. Step 2
+3. ...
+
+### Expected behavior
+A clear and concise description of what you expected to happen.
+
+### System
+
+- OS: [e.g. MacOS 10.14.3]
+- Terminal: [e.g. iTerm, PowerShell]
```
Comment (terkelg) [inline .github/bug_report.md]: Terminal is not relevant in this context and should be removed

**Comment by cliffordfajardo** [comment]
@terkelg - done. Thanks for letting me contribute lol 💯 🤘, I know how easy it could have been for you have done this.

**Pull request merged**